tests/unit_tests/blockchain_db.cpp

```diff
@@ -242,27 +242,31 @@ TYPED_TEST_CASE(BlockchainDBTest, implementations);
 
 TYPED_TEST(BlockchainDBTest, OpenAndClose)
 {
-  std::string fname(tmpnam(NULL));
+  boost::filesystem::path tempPath = boost::filesystem::temp_directory_path() / boost::filesystem::unique_path();
+  std::string dirPath = tempPath.string();
 
-  this->set_prefix(fname);
+  this->set_prefix(dirPath);
```

Comment:
No prob. 

Also, note that the recommended replacement would not have been **mkstemp**, but ** mkdtemp**, which prompted me to rename the variable for clarity, even though I used boost instead.